### ISSUE TITLE
IoT - Example page with temperature data

### ISSUE THREAD

**Issue by fyziktom** [fyziktom/VirtualEconomyFramework#40]
We need to create simple HTML/JS/CSS page which will display IoT Message NFTs from HARDWARIO CHESTER Device. 

We have example [page here](https://hardwario.veframework.com/chester-temperature/temperature.html?address=NWpT6Wiri9ZAsjVSH8m7eX85Nthqa2J8aY). Source will be pushed to branch related to this issue.

It need UX/UI refactoring. Also the functions to obtain the data is better to refactor and cover in some class to allow possibility to use it in other simple pages like this. 

It has two option how to obtain the NFT data. One is from Neblio API and second get the NFT data from the VENFT App Server. It is good to keep both options, but main focus is direct read of these NFT data, because they using latest state of the NFT.

**Comment by fyziktom** [comment]
I have created branch [40-IoT-Temperature-OnePage](https://github.com/fyziktom/VirtualEconomyFramework/tree/40-IoT-Temperature-OnePage)

And I have uploaded the [example OnePage](https://github.com/fyziktom/VirtualEconomyFramework/tree/40-IoT-Temperature-OnePage/VirtualEconomyFramework/VENFT-JS-OnePages/IoT/HARDWARIO/Temperature)

**Comment by fyziktom** [comment]
I added API command in VENFT App Server which provide parsed values from address NFTs IoT Messages.
It is just in this branch now. The address is be preload by server to save time of load. It has little uncomfortable setting now, but it is kind of hotfix. The APIs likes this will be solved in issue #44  :)

Now, I pushed it for testing purposes to azure. It is possible to test it [here](https://venftappserver.azurewebsites.net/swagger/index.html).

The command is this one:
https://venftappserver.azurewebsites.net/iotapi/GetTemperatures

It is **POST** with these parameters:

```json
{
  "address": "NWpT6Wiri9ZAsjVSH8m7eX85Nthqa2J8aY",
  "start": 0,
  "loadmax": 2
}
```

- Now this test server keeps the data just about this specific address so it is just one you can try to load.
- The "start" tells the offset of start parsing nft from newest.
- The "loadmax" tells how many items should be returned (counted from "start")

This is response for the request with the test parameters:

```json
{
  "address": "NWpT6Wiri9ZAsjVSH8m7eX85Nthqa2J8aY",
  "data": [
    {
      "txid": "6f1abb3c95709556d711535e301373a920f67108e653ebca14151aab9d1e08ee",
      "datetime": "2022-01-19T21:39:03Z",
      "description": "IoT Device Temperature",
      "temperature": 22.12,
      "latitude": 50.7633629,
      "longitude": 15.0550699
    },
    {
      "txid": "29a99b38c5ec0d21d6f5ce62bc2dee38fcd5f82f6efee4b77d7cc533c1a52328",
      "datetime": "2022-01-19T21:24:02Z",
      "description": "IoT Device Temperature",
      "temperature": 22.12,
      "latitude": 50.7633629,
      "longitude": 15.0550699
    }
  ]
}
```

**Comment by schamannnet** [comment]
Ahoj,
Tak jak dopadlo nataceni - libilo se to i na HWio?
mam covid vcera jsem byl uplne vyrizenej, dnes uz se citim trochu lip.
takze to cos udelal tam bohuzel nebylo naimplementovano, budeme to
dodelavat, nebo to bylo pouze prezentacni a dal se s tim uz nepocita?

Dej vedet co dal...

Přeji příjemný den

Mirek Šalamon

Tel: 773400722
mail: ***@***.***

PURE CREW s.r.o.
Rybná 24
Praha 1
111 00

http://purecrew.cz
http://l <http://dispecink.online>okalizace.eu
http://driveandsleep.eu
http://nejlevnejsi-hosting.cz


čt 20. 1. 2022 v 0:53 odesílatel fyziktom ***@***.***> napsal:
